website/docs/timeline.md

```diff
@@ -54,6 +54,35 @@ after certain thresholds the archival kicks in to move older timeline events to
 timeline is never contacted for regular operations of the table and is merely used for book-keeping and debugging purposes.
 Any instants seen under “.hoodie” directory refers to active timeline and those archived goes into “.hoodie/archived” folder.
 
+#### LSM Timeline
+
+As mentioned above, active timeline has limited log history to be fast, while archived timeline is expensive to access
```

Comment:
@codope  Can we clarify in a sentence or two on how users can/should transition to this new timeline. Should there be any action items from the users to use this new timeline. That part is not clear.

Reply:
For the beta release it is enabled by default. Users will be automatically transitioned to the new timeline and can't go back unless they downgrade the hudi version. I can mention about a config which controls the timeline compaction - `hoodie.timeline.compaction.batch.size`.